webpack/react_app/components/RegistrationExtension/RexPull.js

```diff
@@ -48,6 +49,12 @@ const RexPull = ({ isLoading, onChange, pluginValues, configParams }) => (
       {/* eslint-disable-next-line camelcase */
       options(configParams?.host_registration_remote_execution_pull)}
     </FormSelect>
+
+    <Alert type="info" isInline style={{ marginTop: '10px' }}>
+      {__(
+        'Please make sure that Smart Proxy is configured for pull-based remote execution.'
+      )}
+    </Alert>
```

Comment:
Could we show this conditionally to avoid situations like the one below?

![image](https://github.com/user-attachments/assets/2502a7e7-db25-4208-8ab0-1654cf8e52b4)

Reply:
Okay, will add the conditions